commit 77750096308b6dc9462ba72e6975ae1d7dced811
Author: Hugo Posnic <hugo.posnic@protonmail.com>
Date:   Tue Jun 29 23:02:59 2021 +0200

    Update meson.build

diff --git a/meson.build b/meson.build
index 587ef8e..247b52d 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('curtail',  
-          version: '1.1.0',
+          version: '1.2.0',
     meson_version: '>= 0.50.0',
   default_options: [ 'warning_level=2',
                    ],
